backend/routes/websocket.go

```diff
@@ -2,20 +2,46 @@ package routes
 
 import (
 	"fmt"
+	"log"
 	"net/http"
+	"os"
+
+	"encoding/json"
 
 	"github.com/gorilla/websocket"
 
 	"github.com/keep-starknet-strange/art-peace/backend/core"
 )
 
-func InitWebsocketRoutes() {
-	http.HandleFunc("/ws", wsEndpoint)
+type Config struct {
+	Host      string `json:"host"`
+	Port      int    `json:"port"`
+	WebSocket struct {
+		ReadBufferSize  int `json:"ReadBufferSize"`
+		WriteBufferSize int `json:"WriteBufferSize"`
+	} `json:"WebSocket"`
 }
 
-var upgrader = websocket.Upgrader{
-	ReadBufferSize:  1024,
-	WriteBufferSize: 1024,
+func LoadConfig() (Config, error) {
```

Comment:
Hey, this file already exists for parsing backend configs. https://github.com/keep-starknet-strange/art-peace/blob/main/backend/config/backend.go

Can you add the websocket stuff there instead, and use `core.ArtPeaceBackend.BackendConfig.WebSocket.ReadBufferSize` and `core.ArtPeaceBackend.BackendConfig.WebSocket.WriteBufferSize` where the values are needed.
Like was done here for the production flag: https://github.com/keep-starknet-strange/art-peace/blob/2dc9e5bd1240a4246317dc214d3c24e745b272d8/backend/routes/pixel.go#L62